src/components/SearchBar/SearchBar.js

```diff
@@ -41,7 +43,7 @@ const styles = {
   },
   searchContainer: {
     margin: 'auto 16px',
-    width: 'calc(100% - 48px - 32px)' // 48px button + 32px margin
+    flex: 'auto'
```

Comment:
This change looks unrelated. Could you explain why it was done or revert it?

Reply:
See https://github.com/TeamWertarbyte/material-ui-search-bar/commit/3ad5f70f380dd4132e2e01dbcc67110edce1fe79